tests/Feature/LoadMoviesCreateTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class LoadMoviesCreateTest extends TestCase
+{
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_create_movies_page_can_be_loaded()
+    {
+        $response = $this->get('/movies/create');
```

Comment:
What happens here if the user isn't logged in? Will it fail if the user isn't logged in and a 403 response is received?